test/propagation.jl

```diff
@@ -225,4 +246,54 @@ using InteractiveUtils: methodswith
         @test nrms(res, w) ≈ 0.375 atol=1e-2
     end
 
+    @testset "Jet transport propagation and TaylorN serialization" begin
+        # Test integration (Apophis)
+
+        # Dynamical function
+        local dynamics = RNp1BP_pN_A_J23E_J2S_eph_threads!
```

Comment:
Minor aesthetic detail: I think a space is missing here at the beginning

Reply:
Done.